nx-X11/programs/Xserver/hw/nxagent/Keyboard.c

```diff
@@ -883,28 +883,28 @@ N/A
 
         XkbFreeKeyboard(xkb, XkbAllComponentsMask, True);
         xkb = NULL;
-	free(rules);
-	free(model);
-	free(layout);
-	free(variants);
-	free(options);
+        free(rules);
```

Comment:
typo in commit message: 

"Keyboard.c: Fix indention" -> "Keyboard.c: Fix indentation"